weighted_levenshtein/clev.pyx

```diff
@@ -179,12 +179,12 @@ def damerau_levenshtein(
     if transpose_costs is None:
         transpose_costs = unit_matrix
 
-    s1 = str(str1).encode()  
-    s2 = str(str2).encode()  
+    # s1 = str(str1).encode()
+    # s2 = str(str2).encode()
```

Comment:
@RevolutionTech @dsu1995 do any of you remember why we did this? It is breaking tests for this PR, but it also does not feel necessary (str1/str2 should already be unsigned char, so don't see an obvious reason to transform this before passing to `c_damerau_levenshtein`)

Reply:
I remember this being related to Python 2/3 compatibility, but I don't remember much more than that.

It looks like duxan authored this change in https://github.com/duxan/weighted-levenshtein/commit/97428a59b5d9a07965792a323a2ae40d5bc8f634.

@duxan, do you remember?